docs/testing/README.md

```diff
@@ -37,8 +37,7 @@ This allows to keep the state of your vault across reboots and is very close to
 ### Starting the iso in a VM
 
 > [!Important]  
-> Because with a VM, we are very close to the real offline *ephemeral vault* setup, we need to move under `src/scripts/*/`, all scripts that can be executed withing the offline vault VM.  
-> This means scripts should be copied over from the `templates/` folder whenever necessary, and **the new files should be added to git** so that they are picked up by nix when building the iso image.
+> Because with a VM, we are very close to the real offline *ephemeral vault* setup, the orca configuration and actions should reflect what you want to test.
 
 Move to the `src/` subdirectory, and run:
```

Comment:
Properly setup the access rights on the dev private key and start the VM:
```bash
chmod go-rwx testing/root_key
nix run src/
```

Reply:
```
Git will, as you stated, track whether a file is executable or not and flip that bit, but permissions are defaulted the same way any other file is. 
```
So not doable via git, let's add it to the doc !